src/vma/dev/buffer_pool.h

```diff
@@ -52,7 +52,11 @@ inline static void free_lwip_pbuf(struct pbuf_custom *pbuf_custom)
 class buffer_pool
 {
 public:
-	buffer_pool(size_t buffer_count, size_t size, pbuf_free_custom_fn custom_free_function);
+	buffer_pool(size_t buffer_count,
+		size_t size,
+		pbuf_free_custom_fn custom_free_function,
+		alloc_t alloc_func = NULL,
```

Comment:
Do we want to use C++ nullptr?

Reply:
Current buffer_pool(cpp|h) source code uses `NULL`. I would prefer to keep single style inside the same module.
Once the decision has been made to move from `NULL` to `nullptr`, this can be done in a custom pull request.

Reply:
I can do this if you believe this change is required for this pull request.

Reply:
@AlexanderGrissik what is conclusion?

Reply:
Lets keep single style